corehq/form_processor/migrations/0091_auto_20190603_2023.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-06-03 20:23
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+TABLE_NAME = 'form_processor_ledgertransaction'
+INDEX_NAME = 'fp_ledgtrans_formid_0fe795_idx'
```

Comment:
did we change something about length of index names?
i get an error for index name to be less than 30 characters, so i have to make custom one with short forms.

Reply:
not that I'm aware of, you should use whatever index name is suggested by Django

Reply:
I did that and the migration failed to run on my system. i have not set anything differently on my local PG.

Reply:
updated to name as suggested by django